# What does the following error message tell you?

# ArgumentError: wrong number of arguments (1 for 2)
#  from (irb):1:in `calculate_product'
#  from (irb):4
#  from /Users/username/.rvm/rubies/ruby-2.0.0-p353/bin/irb:12:in `<main>'


# The error message says that the calculate_product is called passing only 1 argument instead of the 2 that are required.